pkg/iac-providers/kustomize/v3/testdata/multibases/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+resources:
```

Comment:
@williepaul  here's how the whole directory structure could be handled within kustomize usage itself. This actually gives more control to the user if you think about it.

Let me walk you through the flow :

1. Kustomize will scan this list of resource directories specified, i.e., dev, stage and prod and create a big cumulative yaml file consisting of all the kubernetes resource which have to be generated by this list of directories.
2. We scan this big generated yaml output string and pass it to the policy engine. So that's how every directory is covered. I thinks it will be better to document it somewhere for the user's guidance. Though an experienced kustomize user would grasp this flow easily.
**3. The obvious caveat is the that we're not able to trace a violation back to its corresponding directory.**

Reply:
This I understand; I'm pointing out a difference in behavior between the helm/kustomize support. For helm, we will scan any charts found within a given directory structure. Whatever we decide is the desirable behavior can be determined later. I see good reasons for both ways, though I have a feeling it will just end up being requested as a feature anyway.